### ISSUE TITLE
Support DeadLetterConfig for Lambda

### ISSUE THREAD

**Issue by gschneider-r7** [rapid7/convection#256]
# Description

Lambda model should support DeadLetterConfig: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-deadletterconfig.html

**Comment by erran** [comment]
It should be noted that this has been possible in convection using the generic `property` setter since well before this ticket was made.

**Comment by gschneider-r7** [comment]
To provide an example of what @erran mentioned:

```ruby
property 'DeadLetterConfig', :TargetArn => 'sqs_or_sns_arn'
```

**Comment by gschneider-r7** [comment]
Available in version 2.2.26
